content/chainguard/chainguard-images/about/versions.md

```diff
@@ -58,6 +58,35 @@ The table provides some example scenarios to help illustrate our approach.
 
 > **Note**: The "Maintained Upstream Releases" column is current as of December 2024.
 
+
+## EOL Grace Period
+
+There are cases where an organization may want to continue using an image after it has reached end of life (EOL). This could be because an image reaches EOL before the organization's release schedule, or perhaps later image versions have one or more issues that prevent the organization from upgrading.
+
+To help in situations like this, Chainguard offers an end-of-life grace period for eligible images. By purchasing a grace period, customers can retain access to versions of images whose primary package has entered its end-of-life phase for up to six months after they have reached EOL.
+
+During this time, Chainguard will address vulnerabilities and update any non-EOL packages within the image (other than the image's primary package). We will continue to build the image and its underlying packages as long as there are no unaddressable critical or high CVEs in the unguarded package itself. 
+
+Chainguard will continue doing so  for a maximum of six months after the primary package enters its EOL phase. Additionally, if an image fails to build because underlying dependencies conflict with the primary package, it will no longer be supported. A failed build signals the end of support for that image. 
+
+As of this writing, the EOL grace period covers approximately 575 container images. To be eligible, images must meet two key requirements:
+
+1. Have multiple version streams
```

Comment:
do we have a link to what "version streams" means? Otherwise maybe describe it with an example?

Reply:
I'm just going to say "multiple release tracks" and link to earlier in the doc for internal consistency